extensions/audio/build.gradle

```diff
@@ -0,0 +1,46 @@
+group 'ai.djl.audio'
+
+dependencies {
+    api project(":api")
+    api project(":basicdataset")
+    api ("org.bytedeco:javacv:1.5.7") {
```

Comment:
Why do we need Java CV for audio?

Reply:
Technically, we only need ffmpeg, but javacv provided a few wrapper class.